Content.Server/Research/Systems/ResearchSystem.Technology.cs

```diff
@@ -23,7 +23,7 @@ public void Sync(EntityUid primaryUid, EntityUid otherUid, TechnologyDatabaseCom
 
         Dirty(primaryUid, primaryDb);
 
-        var ev = new TechnologyDatabaseModifiedEvent();
+        var ev = new TechnologyDatabaseSynchronizedEvent();
         RaiseLocalEvent(primaryUid, ref ev);
```

Comment:
This is a breaking change and should be listed on the PR please.

Reply:
Derp, I originally wasn't solving it this way and forgot to go back and add one. Thanks!